lib/transfer.js

```diff
@@ -22,11 +22,11 @@ define('transfer', function (require) {
 		   # @Parameters: AccountID
 		   # @Returns an array of JSON Objects containing the transfers for that account.
 		**/
-		getAll: function(accID) {
+		getAll: function(accID, type) {
```

Comment:
Add the ability to not include type if the user wants to get BOTH the payer and payee transfers.